XRTK-Core/Packages/com.xrtk.core/Definitions/Controllers/Simulation/SimulatedControllerDataProviderProfile.cs

```diff
@@ -0,0 +1,227 @@
+﻿// Copyright (c) XRTK. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections.Generic;
+using UnityEngine;
+using XRTK.Attributes;
+using XRTK.Definitions.Controllers.Simulation.Hands;
+using XRTK.Definitions.Utilities;
+using XRTK.Interfaces.Providers.Controllers.Simulation;
+
+namespace XRTK.Definitions.Controllers.Simulation
+{
+    [CreateAssetMenu(menuName = "Mixed Reality Toolkit/Input System/Controller Data Providers/Simulated Controller Data Provider Profile", fileName = "SimulatedControllerDataProviderProfile", order = (int)CreateProfileMenuItemIndices.Input)]
+    public class SimulatedControllerDataProviderProfile : BaseMixedRealityControllerDataProviderProfile
+    {
+        #region General Settings
+
+        [Header("General Settings")]
+
+        [SerializeField]
+        [Tooltip("Enable controller simulation")]
+        private bool controllerSimulationEnabled = false;
+
+        /// <summary>
+        /// Is hand simulated hand tracking enabled?
+        /// </summary>
+        public bool ControllerSimulationEnabled => controllerSimulationEnabled;
+
+        [SerializeField]
+        [Implements(typeof(IMixedRealitySimulatedController), TypeGrouping.ByNamespaceFlat)]
+        [Tooltip("The concrete controller type to simulate.")]
+        private SystemType simulatedControllerType = null;
+
+        /// <summary>
+        /// The concrete controller type to simulate.
+        /// </summary>
+        public SystemType SimulatedControllerType
+        {
+            get => simulatedControllerType;
+            private set => simulatedControllerType = value;
```

Comment:
Do we actually set this someplace? Do we need the setter?

Reply:
No we don't. Good find. I'll remove it.